src/main/java/co/rsk/federate/btcreleaseclient/BtcReleaseClient.java

```diff
@@ -295,9 +292,13 @@ public void onBlock(org.ethereum.core.Block block, List<TransactionReceipt> rece
          */
         private boolean isSVPSpendTxReadyToSign(long currentBlockNumber, Map.Entry<Keccak256, BtcTransaction> svpSpendTx) {
```

Comment:
Btw, the comment above is out of date. No `svpTxHash` param anymore

Reply:
Updated